src/decompiler.cc

```diff
@@ -230,17 +238,23 @@ struct Decompiler {
                struc.empty() ? GetDecompTypeName(t) : struc);
   }
 
-  bool ConstIntVal(const Expr* e, uint64_t &dest) {
+  bool ConstIntVal(const Expr* e, uint64_t& dest) {
     dest = 0;
-    if (!e || e->type() != ExprType::Const) return false;
+    if (!e || e->type() != ExprType::Const)
+      return false;
```

Comment:
Revert or add braces.. it looks like this pattern is only common in this one file, so adding braces is probably the way to go here and elsewhere in this file.